app/src/main/java/org/mozilla/tv/firefox/FirefoxApplication.kt

```diff
@@ -89,18 +90,14 @@ open class FirefoxApplication : LocaleAwareApplication() {
     }
 
     private fun initPush() {
-        var admAvailable = false
-        try {
-            Class.forName("com.amazon.device.messaging.ADM")
-            admAvailable = true
-        } catch (e: ClassNotFoundException) {
-            android.util.Log.i(DEFAULT_LOGTAG, "ADM is not available on this device.")
-        }
-
+        var admAvailable = ADM(applicationContext).isSupported
         // Only install push feature if ADM is available on this device
         if (admAvailable) {
+            Log.log(tag = DEFAULT_LOGTAG, message = "ADM not available")
```

Comment:
The logger is intended to be used like https://groups.google.com/a/mozilla.com/forum/#!search/android$20logger/amazon-dev/6BIH4epQGU8/6AoGkzt5BAAJ

Also, the android Log statement below should be replaced with the a-c logger too.